client.go

```diff
@@ -495,7 +503,10 @@ func (client *Client) worker() {
 // send's success.
 func (client *Client) Capture(packet *Packet, captureTags map[string]string) (eventID string, ch chan error) {
 	if client == nil {
```

Comment:
Oops sorry, this was a weird copy-and-paste issue; clearly I didn't review my own commit oops